### ISSUE TITLE
Eat 0.9 terminal supports Sixel

### ISSUE THREAD

**Issue by sfllaw** [hpjansson/chafa#197]
The Eat terminal supports sixels: https://www.reddit.com/r/emacs/comments/172vgex/eat_09_released/

It looks like the proper way to detect it is to look for the `EAT_SHELL_INTEGRATION_DIR` environment variable: https://codeberg.org/akib/emacs-eat/src/commit/3a6f418f55d183b9d86f99c140caed4ba3d44f93/eat.el#L6993-L6994

The other alternative is to look for `TERM` that starts with `eat-`: https://codeberg.org/akib/emacs-eat/src/commit/3a6f418f55d183b9d86f99c140caed4ba3d44f93/eat.el#L4800-L4812

If you like, I can prepare a patch. It looks like it would go into `detect_capabilities`? https://github.com/hpjansson/chafa/blob/d3bc4e765862070f2560bd396e8366298b16ae12/chafa/chafa-term-db.c#L489-L492

**Comment by hpjansson** [comment]
Yes, I'd be happy to take a PR! It's sufficient to check TERM (the same way as foot-), but if you also check EAT_SHELL_INTEGRATION_DIR, it may work better inside tmux/screen too.

**Comment by sfllaw** [comment]
@hpjansson I have implemented this in #198, which is ready for review.

**Comment by hpjansson** [comment]
Eat looks interesting, I'll have to try it out. With a good terminal inside Emacs, my world would be complete.
